pkg/framework/describe.go

```diff
@@ -30,3 +30,7 @@ func BuildSuiteDescribe(text string, body func()) bool {
 func ClusterRegistrationSuiteDescribe(text string, body func()) bool {
 	return Describe("[cluster-registration-suite "+text+"]", Ordered, body)
 }
+
+func ReleaseStrategyDescribe(text string, body func()) bool {
```

Comment:
Based on the other function names here, maybe this should be `ReleaseSuiteDescribe` instead of `Strategy`?